syz-ci/syz-ci.go

```diff
@@ -265,6 +271,34 @@ func main() {
 	}
 }
 
+func deprecateAssets(cfg *Config, shutdownPending chan struct{}) {
+	dash, err := dashapi.New(cfg.DashboardClient, cfg.DashboardAddr, cfg.DashboardKey)
+	if err != nil {
+		log.Logf(0, "deprecateAssets: failed to create dashapi: %v", err)
+		return
+	}
+	storage, err := asset.StorageFromConfig(cfg.AssetStorage, dash)
+	if err != nil {
+		log.Logf(0, "deprecateAssets: failed to create asset storage: %v", err)
+		return
+	}
+	rand.Seed(time.Now().UTC().UnixNano())
+loop:
+	for {
+		sleepHours := 6 + rand.Intn(6)
+		select {
+		case <-shutdownPending:
+			break loop
+		case <-time.After(time.Hour * time.Duration(sleepHours)):
+		}
+		log.Logf(0, "running deprecateAssets()")
```

Comment:
In Go function names are not generally followed by (), it's not really necessary and does not add much info. That's typically done in C.
Also we tend to not include function names in log messages:
https://gist.githubusercontent.com/dvyukov/ad3957446f8a9239b4c1ad4f0575316b/raw/2161ea2e644b65659e2be42879ff6838c9e6b43b/gistfile1.txt
but rather say something more in user terms (e.g. "deprecating assets...").